modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/simulation/v0.1.0.json

```diff
@@ -0,0 +1,64 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/simulation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:DetailedCircuitSimulationShape",
```

Comment:
I think there is nothing here that is specific to simulations of detailed circuits. The same model could be used for simulations of simplified circuits or for simulations of single neurons, of biochemical reactions, etc.

Reply:
Agreed. Let use SimulationShape then. @genric made the change. It will be merged.